textproto/mod.ts

```diff
@@ -64,14 +60,14 @@ export class TextProtoReader {
     let buf = await this.r.peek(1);
     if (buf === null) {
       return null;
-    } else if (buf[0] == charCode(" ") || buf[0] == charCode("\t")) {
```

Comment:
Shouldn't this be `buf[0] == CHAR_SPACE || buf[0] == CHAR_TAB`?

Ditto for lines 74 and 109

Reply:
Excellent catch, I even improved this part by creating a constant as this expression is used repeatedly.